gnpy/core/equipment.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+'''
+nf model parameters calculation
+calculate nf1, nf2 and Delta_P of a 2 coils edfa with internal VOA
+from nf_min and nf_max inputs 
+'''
+import numpy as np
+from gnpy.core.utils import lin2db, db2lin, load_json
+
+GAIN_MIN_FIELD = "gain_min"
```

Comment:
Unless someone is going to change this at runtime, I don't think we gain anything from using a constant here.

It is not uncommon for Python code to use strings as constants.

In this particular case, you're basically just trading a `NameError` for a `KeyError`.

If you like the use of enumerations, there is https://docs.python.org/3/library/enum.html

However, in this case, field keys are generally just strings.

Reply:
?
These are just Cte to record the json field names, rather than hard coding in the middle of the code. This way if we change the attributs in the json eqpt file, it's easier to know what to modify in the code.

Reply:
It's fine to use hardcoded strings in this fashion.

Consider that the use a variable here adds a non-semantic, non-abstracted indirection, which is evident from the variable name being identical to its value.